applications/portal/backend/api/services/search_service.py

```diff
@@ -6,25 +6,27 @@
 from api.repositories import search_repository
 from openapi.models import FilterRequest
 from openapi.models import Antibody as AntibodyDTO, PaginatedAntibodies
-from ..mappers.antibody_mapper import AntibodyMapper
 from . import antibody_service
 
-antibody_mapper = AntibodyMapper()
 
 
-def fts_antibodies(page: int = 1, size: int = 50, search: str = '') -> List[AntibodyDTO]:
+def fts_antibodies(page: int = 1, size: int = 10, search: str = '', filters=None) -> List[AntibodyDTO]:
     page = page or 0  # required at the moment, issue with default values in main.py-L134
-    size = size or 50  # required at the moment, issue with default values in main.py-L134
+    size = size or 10  # required at the moment, issue with default values in main.py-L134
+    return fts_and_filter_antibodies(page=page, size=size, search=search, filters=filters)
+    
+def filter_antibodies(body: FilterRequest) -> List[AntibodyDTO]:
```

Comment:
The name "body" is not great in the service layer. It's indicating a web request, while the service layer must be all about business logic, hence completely agnostic of the call being coming from a POST payload or anything else